plugins/main/public/utils/applications.ts

```diff
@@ -897,12 +897,20 @@ export const Categories = [
     order: 500,
     euiIconType: 'watchesApp',
   },
+  {
+    id: 'wz-category-agents-management',
+    label: i18n.translate('wz-app-category-agents-management', {
+      defaultMessage: 'Agents management',
+    }),
+    order: 600,
+    euiIconType: 'indexRollupApp',
+  },
   {
     id: 'wz-category-server-management',
     label: i18n.translate('wz-app-category-server-management', {
       defaultMessage: 'Server management',
     }),
-    order: 600,
+    order: 700,
```

Comment:
**nitpick**: the `order` of some categories were changed. According to the application order convention (https://github.com/wazuh/wazuh-dashboard-plugins/blob/1e106ace081d9821ad1b350dfb1a9214c1bd2412/plugins/main/public/utils/applications.ts#L10-L30), they should have an `order` related to the category they belong to.

Reply:
Fixed in [commit]([4c7b306](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7112/commits/4c7b3060debaeacbd35a9d64a76e3ea11b73183c))